### PR TITLE
Add buttons on watches sidebar

### LINKED ISSUE #174: BUG: Check ticks above search panel
I'm running with Ubuntu 14.04 and atom 1.0.5 and see the tick marks above the search panel:

bug_hydrogen

### PR DESCRIPTION
**Pull request by lgeiger**
I added buttons to the watches sidebar to trigger the `toggle watches` and `remove watch` commands. What do you think about the styling?

old vs new:

old new

The possibility to copy one line outputs by clicking on them was added as well:

copy

It fixes #174 with this hack https://github.com/atom/atom/issues/6857

Apart from that this PR fixes some styling issues mostly related to scrollbars by changing `overflow` from `scroll` to `overlay`. Can you test if it works on Windows and Linux as well?

### CODE REVIEW

**Comment by slavaGanzin** [comment]
Would be great if you add command for copying result too. 
Styles look great.

If your interested in watches functionality additionally I ask you too invent some interface for this https://github.com/nteract/hydrogen/pull/143

**Comment by lgeiger** [comment]
> Would be great if you add command for copying result too.

I don't know what you mean?

> I ask you too invent some interface for this #143 

I'll take a look at this.
But I must say I quite like the scrollbar history ;)

**Comment by slavaGanzin** [comment]
I'm talking about copying result with hydrogen command which can be mapped to some keybindings.
> But I must say I quite like the scrollbar history ;)

Does it looks intuitive for you? If so we shouldn't reinvent wheel

**Comment by rgbkrk** [comment]
@slavaGanzin do you think we could merge this PR and you could address the review feedback in a follow on PR?

File: styles/hydrogen.less [hunk 1]
```diff
@@ -79,7 +83,8 @@
 
         .bubble-output-container {
             padding: 3px;
-            overflow: scroll;
+            padding-right: 10px;
+            overflow: auto;
         }
```
Comment (slavaGanzin) [inline styles/hydrogen.less]: Is scrolling available with overflow: auto?

**Comment by slavaGanzin** [comment]
It's all about view. If this https://github.com/lgeiger/hydrogen/commit/e5b061b54551cac6c3758c50fb5a0baea49b110a were separated this PR could be merged even without review. Views don't broke logic.

I think we should merge this.

**Comment by slavaGanzin** [comment]
@lgeiger Thanks for your work

**Comment by rgbkrk** [comment]
Thank you @lgeiger!

File: styles/hydrogen.less (hunk 1 above)
Comment (lgeiger) [inline styles/hydrogen.less]: Yes scrolling works.
For me the only difference is that the space for the scrollbar isn't reserved with `auto` when no scrolling is needed. I'm talking about these dark bars:
bildschirmfoto 2016-02-26 um 11 58 28

**Comment by lgeiger** [comment]
For me the scrollbar is fine. The only issue is that on Mac the default is that scrollbars don't show when you work with a trackpad. Do you know if there is an option in Atom to force the scrollbars?

In the future I'll split my PR in a more logical way and don't try to squash everything in one. Sorry :)

@slavaGanzin About C++: I work with [ROOT](https://root.cern.ch/) they have a [Jupyter Kernel](https://github.com/root-mirror/root/tree/master/bindings/pyroot/JupyROOT) and it works quite well.

**Pull request merged**